pkgs/misc/cups/drivers/brother/mfc6490cw/default.nix

```diff
@@ -0,0 +1,115 @@
+{ pkgsi686Linux
+, stdenv
+, a2ps
+, coreutils
+, dpkg
+, fetchurl
+, file
+, ghostscript
+, gnugrep
+, gnused
+, lib
+, makeWrapper
+, perl
+, psutils
+, which
+}:
+
+let
+  model = "mfc6490cw";
+  version = "1.1.2-2";
+  reldir = "usr/local/Brother/Printer/${model}";
+  filterFile = "lpd/filter${model}";
+
+in rec {
+  driver = pkgsi686Linux.stdenv.mkDerivation rec {
+    inherit version;
+    src = fetchurl {
+      url = "https://download.brother.com/welcome/dlf006180/${model}lpr-${version}.i386.deb";
+      sha256 = "1p33aal3dv3r03v9nbafrzx2bhij34hrsg2d71kwm738a8n18vnj";
+    };
+    name = "${model}drv-${version}";
```

Comment:
```
name = "${model}lpr-${version}"
```

To reflect upstream package name and the name exposed in the all-packages.nix module.